src/main/java/core/basesyntax/controller/BetHandler.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.impl.BetDaoImpl;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class BetHandler implements Handler {
```

Comment:
let's have only one class named ConsoleHandler that will receive all info from console